.gitignore

```diff
@@ -17,3 +17,4 @@ build
 coverage.lcov
 .env/
 .venv/
+.vscode/
```

Comment:
A nit from me here.

Stuff like this (as well as `.DS_Store`) is not strictly related to a Python project (or any other project). That doesn't help users of any other editor/IDE.

A much better place for such entries is the "global" ignore file. Adding those in there will save you from re-adding them to every single project you work on.

`man gitignore | grep -A5 'all situations'` is your friend.